recipes/galaxy-objectstore/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "20.9.1" %}
+{% set sha256 = "3cdc27ddcdf891142a6af0601cf0490fc3c0ecc28e343c140500b7da8403412b" %}
+{% set galaxy_version = version.split(".")[:2]|join(".") %}
+
+package:
+  name: galaxy-objectstore
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/09/5b/26662352bddaf874997fae4014218548bd4eb847eb86f9a0f9a61ffd9f4b/galaxy-objectstore-{{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://files.pythonhosted.org/packages/05/89/b1964bda19f06f497b2179f98a56810a5bf66fc4e72dbfee79a053022fa4/galaxy-objectstore-{{ version }}.tar.gz
  
  
```

Reply:
Those URLs are actually better: https://pypi.io/packages/source/g/galaxy-objectstore/galaxy-objectstore-20.9.1.tar.gz

We can use than the bot to update the recipes :)